apisix/secret/vault.lua

```diff
@@ -26,6 +26,7 @@ local norm_path = require("pl.path").normpath
 local sub        = core.string.sub
 local rfind_char = core.string.rfind_char
 
```

Comment:
We can avoid the blank line here? Other files don't add a blank line among localized variables.